composer.json

```diff
@@ -23,12 +23,14 @@
         "ezyang/htmlpurifier": "^4.10",
         "geoip2/geoip2": "~2.4.2",
         "greenlion/php-sql-parser": "^4.1",
+        "guzzlehttp/cache-subscriber": "~0.2.0",
```

Comment:
Any reason to add this one? it's already in composer.lock because of our dependencies :thinking: